src/oas.md

```diff
@@ -84,6 +84,40 @@ Some examples of possible media type definitions:
   application/vnd.github.v3.patch
 ```
 
+JSON-based and JSON-compatible YAML-based media types can make direct use of the [Schema Object](#schema-object) as the Object uses JSON Schema.
+The use of the Schema Object with other media types is handled by mapping them into the JSON Schema [instance data model](https://www.ietf.org/archive/id/draft-bhutton-json-schema-01.html#name-instance-data-model).
+These mappings may be implicit based on the media type, or explicit based on the values of particular fields.
+Each mapping is addressed where the relevant media type is discussed in this section or under the [Media Type Object](#media-type-object) or [Encoding Object](#encoding-object)
+
```

Comment:
```suggestion
To optimize for either time or space complexity, an implementation applying a schema to content of a mapped media type MAY forgo serializing the data into JSON and running a standard JSON Schema validator provided that the optimized validation logic is manifestly equivalent for all inputs.

```

Reply:
@hudlow OK so _I_ know what this means, but I feel like it's a bit too abstract for most folks based on past feedback.  I feel like addressing just the main case at hand is going to work better, although maybe I'm overcorrecting.   idk.

Reply:
Understood, whatever you think.